Makefile.include

```diff
@@ -730,7 +730,7 @@ debug-server:
 	$(call check_cmd,$(DEBUGSERVER),Debug server program)
 	$(DEBUGSERVER) $(DEBUGSERVER_FLAGS)
 
-emulate:
+emulate: all
```

Comment:
Out of curiosity: why is this needed?

Reply:
When you only call `make emulate` the binary doesn't get compiled. So I added `all` so that the binary gets compiled. Otherwise it fails when you haven't called another target before.

Reply:
That sounds like a good idea :-) I'll cherry-pick that commit in my branch.

Reply:
It's in: https://github.com/basilfx/RIOT/commit/77473003c60df767bfb7d7b57f5c319a5c21c9ee